src/handlers.rs

```diff
@@ -107,6 +108,20 @@ pub async fn handle(ctx: &Context, event: &Event) -> Vec<HandlerError> {
         }
     }
 
+    if let Some(config) = config
```

Comment:
I would expect that this can go in as an issue handler rather than ad-hoc like this?

Reply:
Oops, yeah, didn't notice there are issue handlers. Should be fixed now.